students/esokada/lesson04/trigrams.py

```diff
@@ -0,0 +1,75 @@
+import random
+
+def processtext(filename):
+    """Open the book file, strip it of header/footer, and return a list of words."""
+    with open(filename, "r", encoding = "utf8") as f:
+        trimmedlines = ""
+        startpoint = False
+        for line in f:
+            #Eliminate everything above the Table of Contents, all the chapter headings, and the footer.
+            if "Contents" in line:
+                startpoint = True
+                continue
+            if "CHAPTER" in line:
+                continue
+            if "End of Project Gutenberg's" in line:
+                break
+            if startpoint == True:
```

Comment:
Should this be if startpoint is is True:, or if startpoint:?

Reply:
Right, thanks! I've got to kick this bad habit.  :)